WebViewControl/GlobalSettings.cs

```diff
@@ -25,6 +28,14 @@ public void AddCommandLineSwitch(string key, string value) {
         
         public IEnumerable<KeyValuePair<string, string>> CommandLineSwitches => commandLineSwitches;
 
+        public Color BackgroundColor {
```

Comment:
May you add a summary comment explaining that on `Color.FromArgb(alpha, r, g, b)`,  background colors with alphas different from 255 are not supported by Cef?